### PR TITLE
Fixing version number

### PR DESCRIPTION
**Pull request by wpotrzebowski**
The previous proposal with 5.x.x-alpha1 didn't work with StrictVersion from distutils. The alternative is to use a1 label e.g. 5.0.5a1

### CODE REVIEW

**Comment by wpotrzebowski** [comment]
ready for testing on OSX

**Comment by butlerpd** [comment]
Does this mean that the `dev` nomenclature suggested in #1872 from PEP 440 not resolve properly?

**Comment by wpotrzebowski** [comment]
ready for testing on Windows

**Comment by wpotrzebowski** [comment]
ready for testing on Win

File: docs/sphinx-docs/source/conf.py [hunk 1]
```diff
@@ -79,7 +79,7 @@
 # The short X.Y version.
 version = '5.0'
 # The full version, including alpha/beta/rc tags.
-release = '5.0.5-alpha.1'
+release = '5.0.5a1'
```
Comment (krzywon) [inline docs/sphinx-docs/source/conf.py:82]: The existing docstring is no longer valid and should be changed. Adding a link to the StrictVersion rules should suffice.

http://epydoc.sourceforge.net/stdlib/distutils.version.StrictVersion-class.html

File: src/sas/sasview/__init__.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 from distutils.version import StrictVersion
-__version__ = "5.0.5-alpha.1"
+__version__ = "5.0.5a1"
 StrictVersion(__version__)
```
Comment (krzywon) [inline src/sas/sasview/__init__.py:3]: Same as my other comment.

**Review by krzywon — CHANGES_REQUESTED** [review]
This fixes the loading issue, but the docstrings need to match the functionality or someone might repeat this mistake in the future.

**Comment by wpotrzebowski** [comment]
ready for testing on OSX

**Comment by wpotrzebowski** [comment]
ready for testing on Linux

**Review by krzywon — APPROVED** [review]

**Comment by krzywon** [comment]
I havent looked at why the builds are failing in this case, but everything here looks good now.

**Comment by wpotrzebowski** [comment]
> I havent looked at why the builds are failing in this case, but everything here looks good now.

Yes, builds are still failing. I am trying to look into this.

**Pull request merged**